### PR TITLE
IARRC Lane Following

### LINKED ISSUE #249: Add integration testing for Lane Following
**Note:** Due to the unvalidated nature of this project, it currently lives on the `iarrc-lane-follow` branch. Once testing is completed, it should be merged into `iarrc`
- Add ```ROSTESTS``` for various scenarios

### PR DESCRIPTION
**Pull request by raadkhan**
this resolves #249

### CODE REVIEW

File: src/decision_igvc/CMakeLists.txt [hunk 1]
```diff
@@ -91,22 +70,9 @@ target_link_libraries(gps_manager ${catkin_LIBRARIES} ${sb_utils_LIBRARIES})
...
-    target_link_libraries(lidar-obstacle-test ${catkin_LIBRARIES})
-
-    catkin_add_gtest(vision-decision-test test/vision-decision-test.cpp src/VisionDecision.cpp
-        WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/test)
-    target_link_libraries(vision-decision-test ${catkin_LIBRARIES})
-
     catkin_add_gtest(gps-mover-test test/gps-mover-test.cpp src/GpsMover.cpp)
     target_link_libraries(gps-mover-test ${catkin_LIBRARIES})
 
-    catkin_add_gtest(final-decision-test test/final-decision-test.cpp src/FinalDecision.cpp)
```
Comment (ValRat) [inline src/decision_igvc/CMakeLists.txt]: I don't think these need to be removed.

File: src/firmware/10-snowbots.rules [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Snowbots udev Rules
```
Comment (ValRat) [inline src/firmware/10-snowbots.rules]: Can delete, has been moved to `setup_scripts`.

File: src/firmware/setup_udev_rules.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+######################################################################
+# This script will add udev rules for the sake of persistent device  #
+# naming                                                             #
+######################################################################
+
```
Comment (ValRat) [inline src/firmware/setup_udev_rules.sh]: Can delete, has been moved to `setup_scripts`.

**Comment by ValRat** [comment]
Seems like the roslaunch(?) checker is also failing due to it not finding a specific file

File: src/decision_igvc/CMakeLists.txt (hunk 1 above)
Comment (raadkhan) [reply src/decision_igvc/CMakeLists.txt]: done

File: src/firmware/10-snowbots.rules (hunk 2 above)
Comment (raadkhan) [reply src/firmware/10-snowbots.rules]: done

File: src/firmware/setup_udev_rules.sh (hunk 3 above)
Comment (raadkhan) [reply src/firmware/setup_udev_rules.sh]: done

**Comment by raadkhan** [comment]
`lane_follow_iarrc` no longer a package of Snowflake

**Pull request closed without merging**